tests/unit/fetcher-test.js

```diff
@@ -62,9 +64,36 @@ describe('Fetcher', () => {
       })
     })
 
+  })
+
+  describe('nowOrWhenFetched 2', () => {
+    let fetcher, docuri, rterm, options, userCallback
+
+    beforeEach(() => {
+      docuri = 'https://example.com/newdoc.ttl'
+      rterm = rdf.namedNode('https://example.com/original.ttl')
+      options = {}
+      userCallback = () => {}
+
+      fetcher = new Fetcher(rdf.graph())
+
+      fetcher.load = sinon.stub().resolves({ ok: true, status: 200, statusText: "Dummy stub"})
+    })
+/*
+    it('should invoke userCallback with caught error', done => {
```

Comment:
Doesn't work, or not needed?

Reply:
In the future, you can always change it into `it.skip`. Then it remains part of the test suite, but is explicitly ignored ("xxx passed tests, 1 pending").